### DISCUSSION TITLE
有没有可能下载字体有点繁琐

### DISCUSSION THREAD

**Discussion by AAcike** [atelier-anchor/smiley-sans#114] [Ideas]
是从b站链接过来的，下载字体链接跳转了几个页面，下载方式能简单点就好了，再加上有很多英文着实看不懂（也是我的原因，不懂英文），没有恶意，这是我下载这个字体过程的感受，非常谢谢UP和研发这个字体的人，辛苦了，谢谢

**Comment by yvvt0379** [comment]
先点releases下面圈起来的部分

![image](https://user-images.githubusercontent.com/62426149/217555287-d3ccaf11-2a41-4200-8171-0d7431e19e98.png)

然后点那个zip文件，他就会下载压缩包

![image](https://user-images.githubusercontent.com/62426149/217555555-849de803-56ae-4540-87f9-ad1fe6d9f267.png)

**Comment by AAcike** [comment]
好的呢，麻烦您了呢，祝您一切顺利



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年2月8日(周三) 晚上10:20
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [atelier-anchor/smiley-sans] 有没有可能下载字体有点繁琐 (Discussion #114)




 
先点releases下面圈起来的部分
 

 
然后点那个zip文件，他就会下载压缩包
